bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractFormComponentImpl.java

```diff
@@ -483,14 +486,32 @@ public boolean isEmpty(SerializerProvider provider, String value) {
      * @return {@code Map<String, String>} returns all custom property key value pairs associated with the resource
      */
     private Map<String, String> getCustomProperties() {
+        Map<String, String> customProperties = new HashMap<>();
+        Map<String, String> templateBasedCustomProperties;
+        List<String> excludedPrefixes = Arrays.asList("fd:", "jcr:", "sling:");
+
+        List<String> reservedProperties = ReservedProperties.getNodeProperties();
```

Comment:
@TalmizAhmed Why did your initial CL even have this 😄  This un-necessary computation could have being saved. Good catch @barshat7 !